src/SFA.DAS.EmployerAccounts.Web/ViewModels/OrganisationAgreementViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using SFA.DAS.EmployerAccounts.Models.Account;
+using SFA.DAS.EmployerAccounts.Models.EmployerAgreement;
+namespace SFA.DAS.EmployerAccounts.Web.ViewModels
+{
+    public class OrganisationAgreementViewModel
+    {
+        public long Id { get; set; }
+        public DateTime? ExpiredDate { get; set; }
+        public long? SignedById { get; set; }
+        public string SignedByName { get; set; }
+        public DateTime? SignedDate { get; set; }
+        public long AccountLegalEntityId { get; set; }
+        public AccountLegalEntity AccountLegalEntity { get; set; }
+        public EmployerAgreementStatus StatusId { get; set; }
+        public AgreementTemplate Template { get; set; }
+        public int TemplateId { get; set; }
```

Comment:
Should the TemplateId be on the Temaplte?

Reply:
removed TemplateId.